### PR TITLE
Stop validating presence

### PR DESCRIPTION
**Pull request by O-kasso**
**Problems**
- We were validating presence wrong: when receiving multiple dimensions to validate at once, we were validating the first, and then quitting.
- Models using `measured` also validate presence via Rails, meaning the same validation is happening twice. This leads to duplicate errors when a validation fails.

**Solution**
Instead of fixing, we're removing validation logic from the gem, relying on Rails to validating presence. This will also resolve our duplicate error issues.

**Review**
Basically just deleted a bunch of code (and changed a test to now allow nil value since our models will be responsible for validating that).

### CODE REVIEW

**Comment by O-kasso** [comment]
@MalazAlamir @kmcphillips CR please?

File: lib/measured/rails/validations.rb [hunk 1]
```diff
@@ -16,8 +16,6 @@ def validate_each(record, attribute, measurable)
 
     return unless measurable_unit.present? || measurable_value.present?
 
-    record.errors.add(attribute, message("cannot be blank")) if [measurable_unit.blank?, measurable_value.blank?].any?
```
Comment (kmcphillips) [inline lib/measured/rails/validations.rb]: Makes sense. We can just use rails `presence: true` validation

File: test/validation_test.rb [hunk 2]
```diff
@@ -76,26 +76,11 @@ class Measured::Rails::ValidationTest < ActiveSupport::TestCase
     assert_equal ["Length units singular custom message too"], thing.errors.full_messages
   end
 
-  test "validation presence works on measured columns" do
```
Comment (kmcphillips) [inline test/validation_test.rb]: No need to remove this, or any of the setup. It's good to prove that the existing rails presence validation continues to work.

**Comment by O-kasso** [comment]
@kmcphillips good point, I was a little too trigger happy. Better now?

File: test/validation_test.rb [hunk 3]
```diff
@@ -91,11 +91,6 @@ class Measured::Rails::ValidationTest < ActiveSupport::TestCase
     refute thing.valid?
   end
 
-  test "validation fails if only if the unit is set" do
```
Comment (MalazAlamir) [inline test/validation_test.rb]: I can see why the validation was originally added, but this makes more sense to me. Let AR handle presence validation if it's added in the models, no need to duplicate behaviour. Do you think it would be worth it to write a test for this?

Comment (MalazAlamir) [inline test/validation_test.rb]: looks like as you mentioned there is test coverage for this in `test/active_record_test.rb`. we gucci
